simulator/export/gdb/gdb_interface.cpp

```diff
@@ -84,9 +84,9 @@ SIM_RC sim_load( SIM_DESC sd, const char * prog_name, struct bfd * /* abfd */, i
     return get_sim( sd).load( prog_name) ? SIM_RC_OK : SIM_RC_FAIL;
 }
 
-SIM_RC sim_create_inferior( SIM_DESC sd, struct bfd * abfd, char *const * /* argv */, char *const * /* env */)
+SIM_RC sim_create_inferior( SIM_DESC sd, struct bfd * abfd, const char *const *argv, const char *const *envp )
```

Comment:
I'm afraid you cannot change signature here since it is externally linked to GDB